docs/General/createCIRAConfig.md

```diff
@@ -24,13 +24,14 @@ A CIRA Configuration provides the Remote Provisioning Client (RPC) the required
     | **Password**  | G@ppm0ym   |
 
 
-7. Provide your development system's IP address as the **Common Name** to generate the self-signed certificate by the MPS server.
+7. Provide your development system's IP address as the **Cert. Hostname(CN)** to generate the self-signed certificate by the MPS server.
 
-8. Do not modify the default value of **Auto Load** for the **MPS root certificate format.**
+    !!! important
+        Do not modify the default value of **Auto Load** for the **MPS root certificate format.**
 
-9. Click **Load** under MPS Root Certificate.
+8. Click **Load** under MPS Root Certificate.
 
-10. Click **Create.**
+9. Click **Create.**
```

Comment:
I just noticed a weird sentence below on line 43 (couldn't seem to select it). 

"Profiles also distinguish between activating in:" 

It's weird because we are expecting that the stuff after the colon, the bulleted lists, will complete the sentence. But it doesn't. 

"Profiles also distinguish between activating in Create a Profile in Client Control Mode." Sounds strange.

Just changing the two bullet things to this should fix it, getting rid of "Create a Profile in" each of them.

Client Control Mode (CCM)
Admin Control Mode (ACM)